lib/renderer/callbacks-registry.ts

```diff
@@ -1,16 +1,17 @@
-'use strict'
-
 const v8Util = process.electronBinding('v8_util')
 
-class CallbacksRegistry {
+export class CallbacksRegistry {
+  private nextId: number
+  private callbacks: Record<number, Function>
+
   constructor () {
```

Comment:
We can remove the constructor now we have members above.  We can initialize them there.